code/game/machinery/suit_storage_unit_vr.dm

```diff
@@ -17,3 +17,48 @@
 		SPECIES_ZORREN_FLAT,
 		SPECIES_ZORREN_HIGH
 	)
+
+/obj/machinery/suit_cycler/explorer
+	name = "Explorer suit cycler"
+	model_text = "Exploration"
+	req_access = list(access_pilot)
+	departments = list("Exploration","Pilot")
+
+/obj/machinery/suit_cycler/explorer/initialize()
+	..()
+	species -= SPECIES_TESHARI
```

Comment:
Not just CALL the parent, but RETURN the parent's return value.

Pick either:
`. = ..()`
or
`return ..()`
and put it in this proc. Both will return the parent's return value. This proc will otherwise return null, which is not acceptable to SSatoms. (`.` is the return value of any proc, and you can set it whenever, it will be returned even if you never use any 'return' at the end, it's implied you return `.` if you don't return anything else)